Assets/BossRoom/Scripts/Client/UI/HeroActionBar.cs

```diff
@@ -43,52 +63,17 @@ void OnDisable()
             }
         }
 
-        public void RegisterInputSender(Client.ClientInputSender inputSender)
+        void OnDestroy()
```

Comment:
Question: You have allocated some memory on OnEnable for m_Tooltips, should we release those back to the system, by explicitly setting `m_Tooltips = null` in an OnDisable function?

Reply:
ok